base/random.jl

```diff
@@ -1310,8 +1310,8 @@ julia> randn(rng, Complex128)
 
 julia> randn(rng, Complex64, (2, 4))
 2×4 Array{Complex{Float32},2}:
- -0.349649-0.638457im  0.376756-0.192146im  -0.396334-0.0136413im  -0.585317+0.0778497im
-  0.611224+1.56403im   0.355204-0.365563im  0.0905552+1.31012im    -0.177608+0.261427im
+ -0.349649-0.638457im  0.376756-0.192146im  …  -0.585317+0.0778497im
```

Comment:
maybe one of the recent printing alignment changes made this display slightly wider than before